### ISSUE TITLE
Do titles have to be unique on a site?

### ISSUE THREAD

**Issue by joelanman** [w3c/wcag#2013]
apologies if this has been raised before, I did a search and couldnt find it

The [Failure of Success Criterion 2.4.2](https://www.w3.org/WAI/WCAG21/Techniques/failures/F25) appears to say that duplicate titles on a site is a failure of 2.4.2:

> Example 2
> A site generated using templates includes the same title for each page on the site. So the title cannot be used to distinguish among the pages.

However the page [Providing descriptive titles for Web pages](https://www.w3.org/WAI/WCAG21/Techniques/general/G88) appears to say that that unique titles 'may also be helpful', implying duplicates are not necessarily a failure:

> It may also be helpful for the title to
> Be unique within the site or other resource to which the Web page belongs

**Comment by patrickhlauke** [comment]
copying my answer from the slack question on this:

normatively no, titles don't need to be unique. they need to be sufficiently descriptive to give the meaning/sense/content of the page.

if that happens to be same/similar to another page, as long as it's still accurate/descriptive, that's perfectly kosher

(whether it's ok for your site to have multiple pages on roughly the same subject/topic is another discussion then, but not one that WCAG cares about)

regarding the failure technique: you're missing the nuance in that failure...if the title its exactly the same on every page, then it's clearly a failure...not because you have duplicates, but because it's unlikely the title there sufficiently describes the content/meaning/purpose of each page

**Comment by joelanman** [comment]
The failure says "So the title cannot be used to distinguish among the pages." which is a good point that I agree with, and would be the case for any number of duplicates. If the point is not about duplicates, possibly this line should be changed.

**Comment by fstrr** [comment]
It looks like this discussion is inactive, so I'm going to close it. If it needs to be re-opened, please do.
